docs/hooks/useTimeZoneConversion.md

```diff
@@ -0,0 +1,117 @@
+|      **serverOffset**       |    _property (number)_    | the difference in milliseconds between the wall-clock time in the server time zone and the client time zone.                                                                                                                                                                                                                                                                                  |
+| **getServerZonedISOString** | _method (returns string)_ | method that takes no arguments and returns a string representation of the wall-clock time according to the server in ISO format. For example, if it is 3 February 2023 at 12:00 in the server time zone, `myDate.getServerZonedISOString()` will return `2023-02-03T12:00:00.000`                                                                                                             |
+| **getClientZonedISOString** | _method (returns string)_ | method that takes no arguments and returns a string representation of the wall-clock time according to the browser in ISO format. For example, if it is 3 February 2023 at 12:00 in the server time zone (Africa/Freetown), but we are using our DHIS2 instance in Oslo, there will be a 1-hour time difference, and `myDate.getClientZonedISOString()` will return `2023-02-03T13:00:00.000` |
+
+**NOTE**: Both `fromServerDate` and `fromClientDate` will return a DHIS2Date object corresponding to the _client/browser_ time zone. This is done so that the time encapsulated by the DHIS2Date object is the true time; JavaScript Date objects are ultimately the number of milliseconds since 1 January 1970 UTC. As such, the representation will be browser specific based on the browser's time zone. The same underlying value of `1675425600000` will be interpreted as `Fri Feb 03 2023 13:00:00` in Oslo (Norway), as `Fri Feb 03 2023 12:00:00` in Freetown (Sierra Leone) and as `Sat Feb 04 2023 01:00:00` in Suva (Fiji). In general, it is likely to be less error prone to consistently make comparisons in the browser time zone (after correcting for time zone differences), so `fromServerDate` is the function applicable for must use cases. `fromClientDate` exists primarily to help get a string representation of the server time zone (see "fromClientDate / server time stamp representation" example below). In advanced cases, you may also want to "trick" the browser into thinking that the wall-clock time for the server is actually the wall-clock time for the browser, in this case you can initialize a new date in conjunction with `fromClientDate` (see "fromClientDate / simulating server wall-clock time" example below)
```

Comment:
This is a very long-winded note that is hard to follow; perhaps it needs a little restructuring/rewriting?